tests/e2e/csi_snapshot_basic.go

```diff
@@ -263,17 +300,25 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 		   volume snapshot as the policy is delete
 		10. Cleanup the pvc
 	*/
-	ginkgo.It("Verify snapshot static provisioning through K8s API workflow", func() {
+	ginkgo.It("[tkg-snapshot] Verify snapshot static provisioning through K8s API workflow", func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 		var storageclass *storagev1.StorageClass
 		var pvclaim *v1.PersistentVolumeClaim
 		var pvclaims []*v1.PersistentVolumeClaim
 		var err error
 		var snapshotContentCreated = false
+		var snapshothandle string
+		var snapshotId string
+		var volHandle string
+
+		if vanillaCluster {
+			scParameters[scParamDatastoreURL] = datastoreURL
+		} else if guestCluster {
+			scParameters[svStorageClassName] = storagePolicyName
+		}
```

Comment:
Are we following the recommended way of creating static snapshots?

Reply:
Also can we check if there are stale entries left in SV and GC

Reply:
No, I am reverting it back to its original Block Vanilla Snapshot code. 
We need to use a recommended way as suggested, for that I will be writing a new test case which will be specific for tkg snapshot.

I will roll that out  in next review